desktop_version/README.md

```diff
@@ -54,12 +59,12 @@ cmake ..
 ```
 
 macOS may be fussy about the SDK version. How to fix this is up to the whims of
-however Apple wants to make CMAKE_OSX_SYSROOT annoying to configure and retain
+however, Apple wants to make CMAKE_OSX_SYSROOT annoying to configure and retain
```

Comment:
This is a completely incorrect comma. Did you even read this line? It's intended to be read like:

> How to fix this is up to the whims of however [much] Apple wants to make CMAKE_OSX_SYSROOT annoying to configure and retain ...

Placing the comma there pauses the sentence for no good reason.